deployment/terraform/create.go

```diff
@@ -273,7 +273,7 @@ func (t *Terraform) Create(initData bool) error {
 
 	// Ingesting the DB dump and restoring the Elasticsearch snapshot can
 	// happen concurrently to reduce the deployment's total time
-	errorsChan := make(chan error, 2)
+	errorsChan := make(chan error, 5)
```

Comment:
Should we keep this to 2 and return on error in the goroutines below? I think the bug here is that we don't return after detecting an error, not that the capacity of this buffered channel is 2.

Reply:
I'm happy either way, since we are going to stop enviornment creation I'd say is better to `return` directly.

Reply:
Changed.